### PR TITLE
feat: .add directives folder and add lazyimg dir fearture

### LINKED ISSUE #34: 全局图片懒加载指令及图片展示错误处理
### Clear and concise description of the problem

图片懒加载:优化页面的性能,减少不必要的DOM渲染,自定义指令可全局使用
### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jersonwei**
### Description

Create directives folder and add a lazyImg_dir  by vueuse/core API;in this folder, you can add any dir_fearture; you just need to add a  'v-dir' instruction  on the correspoding HTML when you want to get the correspoding effect  
close #34 

### What is the purpose of this pull request? 

- [ ] Bug fix
- [X] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [X] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [X] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

**Comment by jersonwei** [comment]
Code rafactoring has completed, waiting for suggestions for the next step

File: src/directives/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ *  Coonfigure and register global directives
+ */
+
+import { App } from 'vue'
```
Comment (Hongbusi) [inline src/directives/index.ts:5]: import type ...

Comment (jersonwei) [reply src/directives/index.ts:5]: 我写的时候写了的,好像自动保存给我优化掉了====

File: src/directives/lazyImg.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const authDirective: Directive = {
+  mounted
+}
+
+export function setupLazyImgDirectives (app: App) {
+  app.directive('lazyImg', authDirective)
+}
+
+export default authDirective
```
[the remark this answers is not in the document]
Comment (jersonwei) [reply src/directives/lazyImg.ts:41]: 1.命名可以改一下,
2.方法这个我看了下主要是因为嵌套函数,另外步骤更加细致和语义化的问题,这个问题应该不大
3.图片的话我就用test那张吧,名字还是default

**Review by Hongbusi — APPROVED** [review]
文档记得写一下

**Pull request merged**